src/domain/ecoverse/ecoverse.service.ts

```diff
@@ -73,19 +72,32 @@ export class EcoverseService {
       ecoverse.partners = [];
     }
 
-    if (!ecoverse.tagset) {
-      this.tagsetService.initialiseMembers(ecoverse.tagset);
-    }
+    // Initialise contained singletons
```

Comment:
Initialise contained singletons? What do you mean?

Reply:
Any objects that are contained by this object